### PR TITLE
[PictureInPicture] Sample renewal for S

### PR DESCRIPTION
**Pull request by yaraki**
Change-Id: I56414b2c32af678a6da1ae2249af5cebc54160bb

### CODE REVIEW

File: PictureInPictureKotlin/app/src/main/java/com/example/android/pictureinpicture/MovieActivity.kt [hunk 1]
```diff
@@ -110,18 +107,26 @@ class MediaSessionPlaybackActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.movie_activity)
```
Comment (nic0lette) [inline PictureInPictureKotlin/app/src/main/java/com/example/android/pictureinpicture/MovieActivity.kt:110]: Should this also use ViewBinding?

File: PictureInPictureKotlin/app/src/main/res/drawable/ic_refresh_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (nic0lette) [inline PictureInPictureKotlin/app/src/main/res/drawable/ic_refresh_24dp.xml:1]: nit: Missing license.

File: PictureInPictureKotlin/app/src/main/java/com/example/android/pictureinpicture/MovieActivity.kt (hunk 1 above)
Comment (yaraki) [reply PictureInPictureKotlin/app/src/main/java/com/example/android/pictureinpicture/MovieActivity.kt]: Done

File: PictureInPictureKotlin/app/src/main/res/drawable/ic_refresh_24dp.xml (hunk 2 above)
Comment (yaraki) [reply PictureInPictureKotlin/app/src/main/res/drawable/ic_refresh_24dp.xml:17]: Done

**Comment by yaraki** [comment]
Thank you, Nicole!

**Review by wanghw — APPROVED** [review]
LGTM on the new API usage

**Review by nic0lette — APPROVED** [review]

**Pull request merged**